src/test/java/com/cloudbees/plugins/credentials/casc/CredentialsCategoryTest.java

```diff
@@ -63,4 +78,41 @@ public void setConfig(String config) {
             this.config = config;
         }
     }
+
+    @Test
+    public void exportUsernamePasswordCredentialsImplConfiguration() throws Exception {
+        UsernamePasswordCredentialsImpl usernamePasswordCredentials =
+                new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL,
+                                                    "username-pass",
+                                                    "Username / Password credential for testing",
+                                                    "my-user",
+                                                    "wonderfulPassword");
+
+        SystemCredentialsProvider.getInstance().getCredentials().add(usernamePasswordCredentials);
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        ConfigurationAsCode.get().export(out);
+
+        // UsernamePasswordCredentialsImpl should be jcasc exported as usernamePassword
+        assertThat(out.toString(), Matchers.containsString("usernamePassword:"));
+        assertThat(out.toString(), not(Matchers.containsString("usernamePasswordCredentialsImpl:")));
+    }
+
+    @Test
+    public void exportCertificateCredentialsImplConfiguration() throws Exception {
+        CertificateCredentialsImpl usernamePasswordCredentials =
+                new CertificateCredentialsImpl(CredentialsScope.GLOBAL,
+                                               "credential-certificate",
+                                               "Credential with certificate",
+                                               "password",
+                                               new CertificateCredentialsImpl.UploadedKeyStoreSource(Base64.encode(
+                                                       "Testing not real certificate".getBytes())));
```

Comment:
Use `java.util.Base64` instead.
```suggestion
                                               new CertificateCredentialsImpl.UploadedKeyStoreSource(Base64.getEncoder().encode("Testing not real certificate".getBytes())));
```